app/controllers/warehouse_reports/client_details/last_permanent_zips_controller.rb

```diff
@@ -28,15 +28,12 @@ def index
       end
 
       @clients = population_service_history_source.
-        joins(:client, :enrollment).
+        joins(:client, :enrollment, :project).
+        includes(:client, :enrollment, :project).
         open_between(start_date: @range.start, end_date: @range.end).
         distinct.
         order(first_date_in_program: :asc).
-        pluck(*columns.values).
-        map do |row|
-          Hash[columns.keys.zip(row)]
```

Comment:
cool. can delete `columns` method from this controller? I think there are a few files where the usage is deleted but not the columns method

Reply:
Right, thanks, good catch.